packages/formation/js/accordion.js

```diff
@@ -40,11 +50,12 @@ const addAccordionClickHandler = () => {
     .querySelectorAll('.usa-accordion, .usa-accordion-bordered')
     .forEach(element => {
       element.addEventListener('click', e => {
-        const target = e.target;
+        const target = e.target.closest('.usa-accordion-button');
 
         // Checks whether the button has a click event already assigned to it.
+        // and if it is a .usa-accordion-button.
         // Specifically React Components.
-        if (!target.onclick) {
+        if (target && !target.onclick) {
```

Comment:
Should the `target` variable be renamed to `accordionButton`, since it's not the `event.target` anymore?

Reply:
Yeah that makes sense.